### ISSUE TITLE
Only Grid sample working in Preview 4

### ISSUE THREAD

**Issue by netclectic** [syncfusion/ej2-aspnet-core-blazor-samples#12]
Only the grid sample seems to be working in the current version of this repo, any other sample throws a json parsing exception: 

```
Uncaught (in promise) Error: Newtonsoft.Json.JsonReaderException: Unexpected character encountered while parsing value: {. Path 'item', line 1, position 9.
  at Newtonsoft.Json.JsonTextReader.ReadStringValue (Newtonsoft.Json.ReadType readType) <0x3a18650 + 0x00502> in <5179dd1b79f047d0ac657ed8a725efc1>:0 
  at Newtonsoft.Json.JsonTextReader.ReadAsString () <0x3a0ecc0 + 0x00006> in <5179dd1b79f047d0ac657ed8a725efc1>:0 
  at Newtonsoft.Json.Serialization.JsonSerializerInternalReader.ReadForType (Newtonsoft.Json.JsonReader reader, Newtonsoft.Json.Serialization.JsonContract contract, System.Boolean hasConverter) <0x39f4bf0 + 0x0012a> in <5179dd1b79f047d0ac657ed8a725efc1>:0 
  at Newtonsoft.Json.Serialization.JsonSerializerInternalReader.PopulateObject (System.Object newObject, Newtonsoft.Json.JsonReader reader, Newtonsoft.Json.Serialization.JsonObjectContract contract, Newtonsoft.Json.Serialization.JsonProperty member, System.String id) <0x3a0da40 + 0x00482> in <5179dd1b79f047d0ac657ed8a725efc1>:0 
  at Newtonsoft.Json.Serialization.JsonSerializerInternalReader.CreateObject (Newtonsoft.Json.JsonReader reader, System.Type objectType, Newtonsoft.Json.Serialization.JsonContract contract, Newtonsoft.Json.Serialization.JsonProperty member, Newtonsoft.Json.Serialization.JsonContainerContract containerContract, Newtonsoft.Json.Serialization.JsonProperty containerMember, System.Object existingValue) <0x3a005d8 + 0x002d6> in <5179dd1b79f047d0ac657ed8a725efc1>:0 
  at Newtonsoft.Json.Serialization.JsonSerializerInternalReader.CreateValueInternal (Newtonsoft.Json.JsonReader reader, System.Type objectType, Newtonsoft.Json.Serialization.JsonContract contract, Newtonsoft.Json.Serialization.JsonProperty member, Newtonsoft.Json.Serialization.JsonContainerContract containerContract, Newtonsoft.Json.Serialization.JsonProperty containerMember, System.Object existingValue) <0x39ff988 + 0x000b0> in <5179dd1b79f047d0ac657ed8a725efc1>:0 
  at Newtonsoft.Json.Serialization.JsonSerializerInternalReader.Deserialize (Newtonsoft.Json.JsonReader reader, System.Type objectType, System.Boolean checkAdditionalContent) <0x39f4158 + 0x001aa> in <5179dd1b79f047d0ac657ed8a725efc1>:0 
  at Newtonsoft.Json.JsonSerializer.DeserializeInternal (Newtonsoft.Json.JsonReader reader, System.Type objectType) <0x39efa80 + 0x000ac> in <5179dd1b79f047d0ac657ed8a725efc1>:0 
  at Newtonsoft.Json.JsonSerializer.Deserialize (Newtonsoft.Json.JsonReader reader, System.Type objectType) <0x39ef570 + 0x00010> in <5179dd1b79f047d0ac657ed8a725efc1>:0 
  at Newtonsoft.Json.JsonConvert.DeserializeObject (System.String value, System.Type type, Newtonsoft.Json.JsonSerializerSettings settings) <0x39ef1a0 + 0x00076> in <5179dd1b79f047d0ac657ed8a725efc1>:0 
  at Newtonsoft.Json.JsonConvert.DeserializeObject[T] (System.String value, Newtonsoft.Json.JsonSerializerSettings settings) <0x3a58548 + 0x0000c> in <5179dd1b79f047d0ac657ed8a725efc1>:0 
  at Newtonsoft.Json.JsonConvert.DeserializeObject[T] (System.String value) <0x3a58510 + 0x00006> in <5179dd1b79f047d0ac657ed8a725efc1>:0 
  at Syncfusion.EJ2.RazorComponents.Lists.EjsListView._InvokeSelect (System.Object arg) <0x3a503e0 + 0x00018> in <3f0681dbecbb4b4c897ee88a64c1679a>:0 
  at (wrapper delegate-invoke) System.Action`1[System.Object].invoke_void_T(object)
  at Syncfusion.EJ2.RazorComponents.BaseComponent.Trigger (System.String eventName, System.Object arg) <0x2c7d688 + 0x00020> in <3f0681dbecbb4b4c897ee88a64c1679a>:0 
  at (wrapper managed-to-native) System.Reflection.RuntimeMethodInfo.InternalInvoke(System.Reflection.RuntimeMethodInfo,object,object[],System.Exception&)
  at System.Reflection.RuntimeMethodInfo.Invoke (System.Object obj, System.Reflection.BindingFlags invokeAttr, System.Reflection.Binder binder, System.Object[] parameters, System.Globalization.CultureInfo culture) <0x20dd0e8 + 0x000d2> in <d795ae78e669403e9a0e7a9acf3f68ee>:0 
--- End of stack trace from previous location where exception was thrown ---

  at Microsoft.JSInterop.DotNetDispatcher.InvokeSynchronously (System.String assemblyName, System.String methodIdentifier, System.Object targetInstance, System.String argsJson) <0x2598068 + 0x00222> in <e1118158fde248beb4358ff5c924bd15>:0 
  at Microsoft.JSInterop.DotNetDispatcher.BeginInvoke (System.String callId, System.String assemblyName, System.String methodIdentifier, System.Int64 dotNetObjectId, System.String argsJson) <0x25ac2e8 + 0x0007c> in <e1118158fde248beb4358ff5c924bd15>:0 
    at endInvokeDotNetFromJS (http://localhost:52258/_framework/blazor.webassembly.js:1:6160)
    at Object.invokeJSFromDotNet (http://localhost:52258/_framework/blazor.webassembly.js:1:5676)
    at _mono_wasm_invoke_js_marshalled (http://localhost:52258/_framework/wasm/mono.js:1:169574)
    at wasm-function[1450]:714
    at wasm-function[601]:22743
    at wasm-function[601]:21413
    at wasm-function[601]:21902
    at wasm-function[601]:21413
    at wasm-function[601]:21413
    at wasm-function[601]:21413
endInvokeDotNetFromJS @ blazor.webassembly.js:1
invokeJSFromDotNet @ blazor.webassembly.js:1
_mono_wasm_invoke_js_marshalled @ mono.js:1
(anonymous)
(anonymous)
(anonymous)
(anonymous)
(anonymous)
(anonymous)
(anonymous)
(anonymous)
(anonymous)
(anonymous)
(anonymous)
(anonymous)
(anonymous)
Module._mono_wasm_invoke_method @ mono.js:1
callMethod @ blazor.webassembly.js:1
beginInvokeDotNetFromJS @ blazor.webassembly.js:1
s @ blazor.webassembly.js:1
e.invokeMethodAsync @ blazor.webassembly.js:1
triggerEJEvents @ ejs.introp.js:36
e.notify @ browser.js:33
e.trigger @ browser.js:195
t.setSelectLI @ list-view.js:1146
t.clickHandler @ list-view.js:785
```

**Comment by uwefms** [comment]
The menu does not work. Do it manually
Instead of
http://localhost:52258/Grid/DefaultFunctionalities?theme=material
use 
http://localhost:52258/Schedule/Default?theme=material

to see the schedule

**Comment by ajithr** [comment]
Hi @netclectic / @uwefms ,

We are sorry for the inconvenience caused. We have resolved the reported issue and published the NuGet package [v17.1.44-beta](https://www.nuget.org/packages/Syncfusion.EJ2.AspNet.Core.RazorComponents/17.1.0.44-beta). We have updated the github repository with latest changes. Could you please pull the latest git changes and try again?

Regards,
Ajith R

**Comment by netclectic** [comment]
Working now, thanks.
